profiles/base/package.use.mask

```diff
@@ -6,6 +6,15 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Michał Górny <mgorny@gentoo.org> (2022-02-07)
+# (on behalf of Jannik Glückert <jannik.glueckert@gmail.com>)
+# libstdc++ and libc++ have ABI-incompatible containers,
+# and any system using libc++ without further advanced tweaks
+# is most likely already broken. Still keep the flag around for adventurers.
+# With the rising popularity of system-clang this flag is becoming
+# a bigger and bigger footgun. Other default-* flags have no ABI impact.
```

Comment:
How about:

```suggestion
# libstdc++ and libc++ have different public ABI that can cause fatal
# incompatibilities unless all C++ packages are built with the same
# stdlib.  Mask the flag by default to prevent accidental damage.
```